aio/aio-proxy/aio_proxy/response/formatters/selected_fields.py

```diff
@@ -0,0 +1,27 @@
+from aio_proxy.labels.helpers import valid_fields_to_select
+
+
+def select_fields_to_include(include_fields, response: dict):
+    # Remove unselected fields from response
+    # In case `include_fields` is None, all extra fields are removed
+    if include_fields is None:
+        for field in valid_fields_to_select:
+            if field.lower() in response:
+                del response[field.lower()]
+        return response
+
+    for field in valid_fields_to_select:
```

Comment:
Iteration over `include_fields` would be more efficient as array is necessary smaller or equal to `valid_fields_to_select`

Reply:
In this case, the refactor does not work because we're looking for fields in `valid_fields_to_select` and not in `include_fields`.